Code/PgSQL/rdkit/rdkit.control

```diff
@@ -1,4 +1,4 @@
 comment = 'Cheminformatics functionality for PostgreSQL.'
-default_version = '4.5.0'
+default_version = '4.5.1'
```

Comment:
This feels like more than just a point change.
Maybe bump to 4.6.0?

Reply:
Yes, it would definitely make sense (when I updated the version number I was still thinking it could be a simple fix).